qemu/tests/virtio_console.py

```diff
@@ -974,77 +976,77 @@ def _s4():
 
         # 0s interruption without any measurements
         if params.get('virtio_console_micro_repeats'):
-            error_context.context("Micro interruptions", logging.info)
+            error_context.context("Micro interruptions", test.log.info)
             threads[1].sendidx = acceptable_loss
             for i in range(int(params.get('virtio_console_micro_repeats'))):
                 interruption()
 
-        error_context.context("Normal interruptions", logging.info)
+        error_context.context("Normal interruptions", test.log.info)
         try:
             for i in range(no_repeats):
                 error_context.context("Interruption nr. %s" % i)
```

Comment:
Fix the issue of missing `test.log.info` ?

Reply:
After our discussion, will skip this suggestion as it seems to be the expected behavior, thanks.